lib/routes/cookbook/index.ts

```diff
@@ -0,0 +1,150 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import logger from '@/utils/logger';
+import puppeteer from '@/utils/puppeteer';
+
+export const route: Route = {
+    path: '/',
+    categories: ['blog'],
+    description:
+        'OpenAI Cookbook 提供了大量使用 OpenAI API 的实用指南和示例代码,涵盖了从基础到高级的各种主题,包括 GPT 模型、嵌入、函数调用、微调等。这里汇集了最新的 API 功能介绍和流行的应用案例,是开发者学习和应用 OpenAI 技术的宝贵资源。',
+    maintainers: ['liyaozhong'],
+    radar: [
+        {
+            source: ['cookbook.openai.com/'],
+        },
+    ],
+    url: 'cookbook.openai.com/',
+    handler,
+};
+
+async function handler() {
+    const rootUrl = 'https://cookbook.openai.com';
+    const currentUrl = `${rootUrl}/`;
+
+    const browser = await puppeteer();
+    const page = await browser.newPage();
+
+    await page.setRequestInterception(true);
+    page.on('request', (request) => {
+        request.resourceType() === 'document' ? request.continue() : request.abort();
+    });
+
+    try {
+        await page.goto(currentUrl, {
+            waitUntil: 'domcontentloaded',
+        });
+
+        const response = await page.content();
+        page.close();
+        const $ = load(response);
+
+        let items = $('a[href^="examples/"]')
```

Comment:
```suggestion
        let items = $('[class="min-h-[90vh]"] .grid a')
```
Parsing the `All` section is enough, since those cards are included in `All`